frontend/public/locales/en/public.json

```diff
@@ -1266,11 +1266,12 @@
   "with exceptions": "with exceptions",
   "Any port": "Any port",
   "NetworkPolicy details": "NetworkPolicy details",
-  "Pods accept all traffic by default. They can be isolated via NetworkPolicies which specify a whitelist of ingress rules. When a Pod is selected by a NetworkPolicy, it will reject all traffic not explicitly allowed via a NetworkPolicy. See more details in: <2></2>.": "Pods accept all traffic by default. They can be isolated via NetworkPolicies which specify a whitelist of ingress rules. When a Pod is selected by a NetworkPolicy, it will reject all traffic not explicitly allowed via a NetworkPolicy. See more details in: <2></2>.",
+  "Pods accept all traffic by default. They can be isolated via NetworkPolicies which specify a whitelist of ingress rules. When a Pod is selected by a NetworkPolicy, it will reject all traffic not explicitly allowed via a NetworkPolicy.": "Pods accept all traffic by default. They can be isolated via NetworkPolicies which specify a whitelist of ingress rules. When a Pod is selected by a NetworkPolicy, it will reject all traffic not explicitly allowed via a NetworkPolicy.",
+  " See more details in: <3></3>.": " See more details in: <3></3>.",
```

Comment:
Do we need the extra space at the beginning ? I see that we are adding it manually.

Reply:
The extra space is intentional so that the optional sentence doesn't abut the `.` of the preceding sentence, *and* we've had issues regarding spaces in Chinese (where there are spaces where they are undesirable), so I didn't want to hard code it.